gulpfile.js

```diff
@@ -97,8 +97,8 @@ gulp.task('unify', ['clean'], function () {
     .pipe(standard())
     .pipe(standard.reporter('default', {
       showRuleNames: true,
-      breakOnError: true,
-      quiet: true
+      breakOnError: process.argv[2] !== "--dont-break",
+      quiet: true,
     }))

```

Comment:
can we make a gulp task for linting only? So we can do `gulp lint`?

Reply:
Done!